packages/metal-dom/src/dom.js

```diff
@@ -254,6 +254,31 @@ class dom {
 	}
 
 	/**
+	 * Verifies if the element is able to trigger the Click event,
+	 * simualting browsers behaviour, avoiding event listeners to be called by dom.triggerEvent method.
+	 * @param {Element} node Element to be checked.
+	 * @param {!String} eventName The event name.
+	 */
+	static isAbleToInteract_(node, eventName) {
+		var currElement = node;
+		var isAble = true;
+		var matchesSelector = 'button, input, select, textarea, fieldset';
+
+		if (eventName === 'click') {
+			while (currElement && dom.match(currElement, matchesSelector) && !dom.match(currElement, 'legend')) {
```

Comment:
It shouldn't be possible for an element to match the `button, input, select, textarea, fieldset` selector and also match `legend`, so this second `match` call can probably be removed, right?